Composer/packages/server/src/models/bot/botProject.ts

```diff
@@ -549,6 +549,91 @@ export class BotProject {
     }
   };
 
+  private _replaceDashInTemplateName = async () => {
```

Comment:
@cwhitten here implement a replace method at server, when load a bot, will update all old template name to new naming convention.

Reply:
But here only ensure old bot can be valid. completion renaming did covered at here, for example :
bfdprompt-1234 --->bfdprompt_1234 --//--> TextInput_Prompt_1234
Do you think we should implement a complete replacement here?

Reply:
Yes, I would like to.